### PR TITLE
do not wrap lonesome sqrts

### PR DESCRIPTION
**Pull request by ChristopherChudzicki**
Addresses #283 

I'm surprised we didn't catch this before 🤷‍♂ 

I tested this on my personal version of demo course, but have not pushed to "public" demo course.

I also bumped version number and changelog. (And I found 3 typos. So I've found 3, and you've found 500 of mine 😄 )

### CODE REVIEW

File: MJxPrep.js [hunk 1]
```diff
@@ -291,9 +291,12 @@ if (window.MJxPrep) {
   // Check for the AsciiMath object every 200ms
   var checkExist = setInterval(updateMathJax, 200);
 
-  // set of AsciiMath symbols classified as mover (math-overscript) tags
+  // FORBIDDEN_TO_WRAP includes things we've found problematic to wrap {:...:},
+  // namelu:
```
Comment (jolyonb) [inline MJxPrep.js]: namely?

File: MJxPrep.js [hunk 2]
```diff
@@ -394,7 +398,8 @@ if (window.MJxPrep) {
 
       var front = eqn.slice(0, funcStart)
       var middle = eqn.slice(funcStart, funcEnd)
-      eqn = front + wrapIfNotOverscript(null, middle) + eqn.slice(funcEnd)
+      console.log(middle)
```
Comment (jolyonb) [inline MJxPrep.js]: Probably don't want this left here?

File: MJxPrep.js (hunk 1 above)
Comment (jolyonb) [reply MJxPrep.js]: @ChristopherChudzicki Bump?

File: MJxPrep.js (hunk 2 above)
Comment (jolyonb) [reply MJxPrep.js]: @ChristopherChudzicki Bump?

File: MJxPrep.js (hunk 1 above)
Comment (ChristopherChudzicki) [reply MJxPrep.js]: fixed, sorry slow

File: MJxPrep.js (hunk 2 above)
Comment (ChristopherChudzicki) [reply MJxPrep.js]: fixed

**Pull request merged**